policies/travel-policy.md

```diff
@@ -165,51 +197,53 @@ OpenSSL will reimburse costs incurred for:
 -   Room type should be a Standard Hotel Room. The Traveller may upgrade
     the room at their own expense.
 -   OpenSSL will fund 1 day's accommodation prior to the required travel
-    and 1 day post the required travel time. Refer Travel Class Table
-    for when additional pre-days can be reimbursed.
+    and 1 day post the required travel time.
 -   Use of the minibar, laundry services or other hotel facilities
     (excluding those required for Business purposes e.g. meeting rooms)
     are at the Traveller's own expense.
 
 ### Meals
 
-The amount listed in the **Meal Entitlement Table** will be the maximum
-amount that can be claimed for each day, it is a per diem amount i.e.
-the allowance is inclusive of all meals that occurred on the relevant
-Day.
+The amount listed in the [Meal Entitlement Table] will be the maximum
+amount that can be claimed for each day, the allowance is inclusive of
+all meals that occurred on the relevant Day.
```

Comment:
You changed it, but not quite enough, there's still the confusing "allowance" in there.  Also, The whole part from the comma and on seems redundant.

The paragraph I proposed was:

``` markdown
The amount listed in the [Meal Entitlement Table] will be the maximum amount that can be
claimed for reimbursement of meals for each day.
```

Reply:
Amended suggested paragraph:

``` markdown
The amount listed in the [Meal Entitlement Table] will be the maximum amount that can be
claimed for reimbursement of all meals per relevant day.
```

Reply:
Saw your fix, it's good enough per the goal of removing "allowance"